app/views/registration_commands/_form.erb

```diff
@@ -48,28 +48,36 @@
   </div>
   <div class='form-group'>
     <label class='col-md-2 control-label' for='setup_insights'>
-      <% help = _('If selected, Insights client will be installed and registered on Red Hat family operating systems. It has no effect on other OS families that do not support it.') %>
+      <% help = _('If selected, Insights client will be installed and registered on Red Hat family operating systems. It has no effect on other OS families that do not support it. Default value is defined by `host_registration_insights` global parameter.') %>
       <%= _('Setup Insights') %>
       <a rel="popover" data-content="<%= help %>" data-trigger="focus" data-container="body" data-html="true" tabindex="-1">
         <span class="pficon pficon-info "></span>
       </a>
     </label>
     <div class='col-md-4'>
-      <% insights_opts = options_for_select([[_('Yes'), 'true'], [_('No'), 'false']], params[:setup_insights]) %>
-      <%= select_tag 'setup_insights', insights_opts, class: 'form-control', include_blank: '' %>
+      <%
+        host_param = @host_config_params['host_registration_insights'] ? _('Yes') : _('No')
+        default_opt = [(_("Default (%s)") % host_param), '']
```

Comment:
I'd recommend keeping this to simple "Default". Yes/No can be confusing when people override the configuration on host group / os (or other) levels. Also the wording should change accordingly, I'll add comments to relevant lines. I think pure "Default" is aceptable and less confusing.

Reply:
Yeah but that's what support guys wanted, to have the values in the form, see the  BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1919958

Reply:
@ares I'm for keeping the Yes/No in the default option, we wanted to do it anyway in the React form refactoring PR

Reply:
I don't mind much, but it is nice to know what the default is right away, so I'd go with the new approach.

Reply:
Or the label can be `Inherit from Host parameter (Yes/No)`

Reply:
yup, that make sense, because it is not a default actually.
Btw does this form rerender when we change hostgroup and/or operatingsystem?
If it doesn't, the information would be incorrect as both of them can change the value ....

Reply:
Label changed to `Inherit from Host parameter (Yes/No)`

> Btw does this form rerender when we change hostgroup and/or operatingsystem?

No, only when the form is submitted. I could add some javascript for live-reloading values, but I'm not sure if there is enough time (just to weeks for Sat 6.9)

And there is a refactoring task I'm working on, where the form is going to be completely in react (with the live-reloading values), see https://projects.theforeman.org/issues/31240

Reply:
Good enough for me, lets hope people are not using it very often.